site/docs/providers/xai.md

```
@@ -156,6 +206,12 @@ This is equivalent to setting `base_url="https://us-west-1.api.x.ai/v1"` in the
 
 ### Live Search (Beta)
 
+:::warning Deprecation Notice
+
+xAI has announced that the Live Search API (via `search_parameters`) will be **deprecated by December 15, 2025**. The replacement is the Agent Tools API, which provides enhanced agentic search capabilities. Agent Tools require the Responses API endpoint - see the [Agent Tools API](#agent-tools-api) section for more details.
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)